### PR TITLE
 5420, Remove/Refac assets part 2

### PR DESCRIPTION
**Pull request by nolawi**
Connects #5420 

### Description

1. This PR removes unsued styles, 
2. Merges selectors
3. Adds the `important` scss-rule

### Testing Plan
1. Should test for regression, on Styleguide, Dispatch, Help Pages, Reader, and  Intake views.
2. Certifications `http://localhost:3000/certifications/2367429/check_documents` 

### TODO
~~1, Test worksheet print~~
~~2, Remove display fonts, check with designers~~

### CODE REVIEW

File: app/assets/stylesheets/reader/_reviewer.scss [hunk 1]
```diff
@@ -78,16 +78,6 @@
...
-// TODO: Get rid of the rule in caseflow commons that makes
-// this necessary!
-// scss-lint:disable QualifyingElement
-svg[aria-hidden=true],
-.fa[aria-hidden=true] {
-  display: inline !important;
-}
-// scss-lint:enable QualifyingElement
-
-
```
Comment (nolawi) [inline app/assets/stylesheets/reader/_reviewer.scss]: Mark, doesn't seem like we need this. LMK if i am missing something

File: app/assets/stylesheets/react/_search_bar.scss [hunk 2]
```diff
@@ -48,14 +48,14 @@
 
     &.cf-search-with-internal-text {
       border-right: 0;
-      -webkit-appearance: none !important;
+      -webkit-appearance: searchfield;
```
Comment (nolawi) [inline app/assets/stylesheets/react/_search_bar.scss]: @dannysellers would this work for the effect you were trying to acheive

File: app/assets/stylesheets/_hearings.scss [hunk 3]
```diff
@@ -533,207 +533,3 @@
     margin-top: 0;
   }
 }
-
```
Comment (nolawi) [inline app/assets/stylesheets/_hearings.scss]: Moved as is to separate file

File: app/assets/stylesheets/react/_search_bar.scss (hunk 2 above)
Comment (dannysellers) [reply app/assets/stylesheets/react/_search_bar.scss]: I don't see any difference between using `none` and `searchfield` :\

File: .scss-lint.yml [hunk 4]
```diff
@@ -43,7 +43,7 @@ linters:
   IdSelector:
     enabled: true
   ImportantRule:
-    enabled: false
+    enabled: true
```
Comment (mdbenjam) [inline .scss-lint.yml]: You are a saint. This is great.

File: app/assets/stylesheets/_commons.scss [hunk 5]
```diff
@@ -426,21 +420,26 @@ svg title {
     height: inherit;
     width: inherit;
   }
+
+  .cf-form-textarea {
+    textarea {
+      max-height: 120px;
+    }
```
Comment (nolawi) [inline app/assets/stylesheets/_commons.scss]: new + applies to only textarea in modals.

File: app/assets/stylesheets/reader/_reviewer.scss (hunk 1 above)
Comment (mdbenjam) [reply app/assets/stylesheets/reader/_reviewer.scss]: I'll check. I'm not sure.

Comment (mdbenjam) [reply app/assets/stylesheets/reader/_reviewer.scss]: Can't figure out what we were using it for. I think this is fine.

**Pull request merged**